src/main/kotlin/xyz/block/dap/Dap.kt

```diff
@@ -2,6 +2,13 @@ package xyz.block.dap
 
 import java.util.regex.Pattern
 
+/**
+ * A Decentralized Agnostic Paytag (DAP).
+ *
+ * A DAP consists of
+ * - a `handle` which is a unique identifier for the user within the domain
+ * - the `domain` of the DAP registry.
```

Comment:
Consider using @property for the properties, as shown [here](https://kotlinlang.org/docs/kotlin-doc.html#block-tags).